Swiftfin tvOS/Views/SettingsView/CustomizeViewsSettings/Components/Sections/ItemSection.swift

```diff
@@ -41,6 +41,10 @@ extension CustomizeViewsSettings {
 
                 ListRowMenu(L10n.enabledTrailers, selection: $enabledTrailers)
 
+                /// Enable Refreshing & Deleting Collections
+                if userSession?.user.permissions.items.canManageCollections ?? false {
```

Comment:
Make these `== true` instead of `?? false`, for boolean values I want to be moving towards that instead.